tests/integration/tests/test_mender_configure.py

```diff
@@ -295,3 +297,26 @@ def test_mender_configure_failed_deployment_apply_fails(
 
     device_config = json.loads(result.stdout)
     assert device_config == configuration
+
+
+@pytest.mark.usefixtures("standard_setup_one_rofs_configure_client")
+def test_mender_configure_managed_configuration():
+    # Generate a simple configuration artifact
+    new_configuration = {"key": "value"}
+    configuration_artifact = tempfile.NamedTemporaryFile(suffix=".mender", delete=False)
+    configuration_artifact_name = configuration_artifact.name
+
+    make_configuration_artifact(
+        new_configuration, "configuration-artifact", configuration_artifact_name
+    )
+
+    deploy.upload_image(configuration_artifact_name)
+    devices = list(
+        set([device["id"] for device in devauth.get_devices_status("accepted")])
+    )
+    deployment_id = deploy.trigger_deployment(
```

Comment:
I was envisioning that this could perhaps be done using the configure endpoint(s) in the backend, to generate an on-the-fly artifact. The way you have done it is equally valid though, after all it is the module we are testing here. Consider it a suggestion only. :-)

Reply:
will do, I'm doing it incrementally, trying to have green build between the various pushes.

Reply:
This is done, but it fails because of https://github.com/mendersoftware/deployments-enterprise/pull/324